crates/next-custom-transforms/src/transforms/react_server_components.rs

```diff
@@ -291,6 +292,10 @@ fn report_error(app_dir: &Option<PathBuf>, filepath: &str, error_kind: RSCErrorK
             RSCErrorKind::NextRscErrInvalidApi((source, span)) => (
                 format!("\"{source}\" is not supported in app/. Read more: https://nextjs.org/docs/app/building-your-application/data-fetching\n\n"), span
             ),
+
+            RSCErrorKind::NextRscErrDeprecatedApi((source, span)) => (
+                format!("\"{source}\" is deprecated."), span
```

Comment:
Can you make sure the error is the same as the runtime error: https://github.com/vercel/next.js/blob/8662ba052a34065a523e71d060a8af3ba320b6f6/packages/next/src/server/web/spec-extension/image-response.ts#L7